.maintain/frame-weight-template.hbs

```diff
@@ -0,0 +1,107 @@
+{{header}}
```

Comment:
This is copied (without modifications) from https://github.com/paritytech/substrate/blob/polkadot-v0.9.37/.maintain/frame-weight-template.hbs .
I don't think we need to touch it to add any notices if its unmodified...?